irteus/Makefile

```diff
@@ -114,24 +114,29 @@ $(LIBIRTEUSIMG): $(INSTALLLIBDIR)/jpegmemcd.$(LSFX) $(IRTEUSIMGOBJS) $(IRTIMGCOB
 $(LIBIRTEUSGL): $(IRTEUSGLOBJS)
 	$(LD) $(SOFLAGS) $(OUTOPT)$(LIBIRTEUSGL) $(IRTEUSGLOBJS) $(IMPLIB)
 
-$(IRTEUSOBJS):
-	$(BINDIR)/eus2$(ESFX) "(setq *objdir* \"$(INSTALLOBJDIR)/\")" < ./compile_irt.l > /dev/null
+$(IRTEUSOBJS): $(OBJDIR)/compile_irt.log
+$(OBJDIR)/compile_irt.log:
+	$(BINDIR)/eus2$(ESFX) "(setq *objdir* \"$(INSTALLOBJDIR)/\")" < ./compile_irt.l > $(OBJDIR)/compile_irt.log
 	-rm -f $(IRTEUS_C) $(IRTEUS_H)
 
-$(IRTEUSGOBJS):
-	$(BINDIR)/eusgl$(ESFX) "(setq *objdir* \"$(INSTALLOBJDIR)/\")" < ./compile_irtg.l > /dev/null
+$(IRTEUSGOBJS): $(OBJDIR)/compile_irtg.log
```

Comment:
After this commit, build behavior changed because of change of build dependency.

- Reproduce the problem
```
$ cd ~/ros/indigo_parent/src/Euslisp/jskeus/irteus
$ touch irtmodel.l
$ make
make: Nothing to be done for `all'.
```

- Expected behavior
compile_irtg.l is called invoked by irtmodel.l timestamp update